src/services/poolServices/index.ts

```diff
@@ -461,3 +466,81 @@ export const checkWithdrawPoolLiquidityGasFee = async (
     payload: partialFee.toHuman(),
   });
 };
+
+export const createPoolCardsArray = async (
+  api: ApiPromise,
+  dispatch: Dispatch<PoolAction>,
+  pools: any,
+  selectedAccount?: InjectedAccountWithMeta
+) => {
+  const apiPool = api as ApiPromise;
+
+  try {
+    const poolCardsArray: PoolCardProps[] = [];
+
+    await Promise.all(
+      pools.map(async (pool: any) => {
+        const lpTokenId = pool?.[1]?.lpToken;
+
+        let lpToken = null;
+        if (selectedAccount?.address) {
+          const lpTokenAsset = await apiPool.query.poolAssets.account(lpTokenId, selectedAccount?.address);
+          lpToken = lpTokenAsset.toHuman() as LpTokenAsset;
+        }
+
+        if (pool?.[0]?.[1]?.interior?.X2) {
+          const poolReserve: any = await getPoolReserves(
+            apiPool,
+            pool?.[0]?.[1]?.interior?.X2?.[1]?.GeneralIndex?.replace(/[, ]/g, "")
+          );
+
+          if (poolReserve?.length > 0) {
+            const assetTokenMetadata: any = await apiPool.query.assets.metadata(
+              pool?.[0]?.[1]?.interior?.X2?.[1]?.GeneralIndex?.replace(/[, ]/g, "")
+            );
+
+            const assetTokenBalance = formatDecimalsFromToken(
+              poolReserve?.[1]?.replace(/[, ]/g, ""),
+              assetTokenMetadata.toHuman()?.decimals
+            );
+
+            const nativeTokenBalance = formatDecimalsFromToken(poolReserve?.[0]?.replace(/[, ]/g, ""), "12");
+
+            poolCardsArray.push({
+              name: `WND–${assetTokenMetadata.toHuman()?.symbol}`,
+              lpTokenAsset: lpToken ? lpToken : null,
+              lpTokenId: lpTokenId,
+              assetTokenId: pool?.[0]?.[1]?.interior?.X2?.[1]?.GeneralIndex?.replace(/[, ]/g, ""),
+              totalTokensLocked: {
+                nativeToken: nativeTokenBalance.toFixed(3),
+                nativeTokenIcon: NativeTokenIcon,
+                assetToken: assetTokenBalance.toFixed(3),
+                assetTokenIcon: AssetTokenIcon,
+              },
+            });
+          }
+        }
+      })
+    );
+
+    poolCardsArray.sort(function (a, b) {
```

Comment:
Instead of 
if (a.name < b.name) {
        return -1;
      }
      if (a.name > b.name) {
        return 1;
      }
      return 0;
    u can use a.name.localeCompare(b.name)